CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+##Contributing to this repository
+
+Welcome!  Thanks for your interest in contributing to Software Carpentry's introductory shell lesson.  
+
+###Intro
+
+Software Carpentry is an open source project, and we welcome contributions of all kinds: new and improved lessons, bug reports, and small fixes to existing material are all useful.
+
+By contributing, you are agreeing that Software Carpentry may redistribute your work under these licenses.
+
+###Making and Submitting Changes
+
+Don't know where to start?  Please see the current list of issues (linked in the right sidebar) for ideas for contributing to this repository. 
+
+For making your contribution, we use the GitHub flow, which is nicely explained in the chapter [Contributing to a Project in Pro Git by Scott Chacon](http://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project).  For this repository, you should branch from and then submit a pull request against the `gh-pages` branch instead of `master`.  
```

Comment:
Maybe

~~~
you should branch from and then submit a pull request against the `gh-pages` branch.
~~~